Xwt/Xwt/CursorType.cs

```diff
@@ -53,13 +53,17 @@ internal CursorType (string id)
 		public static readonly CursorType ResizeUp = new CursorType ("ResizeUp");
 		public static readonly CursorType ResizeDown = new CursorType ("ResizeDown");
 		public static readonly CursorType ResizeUpDown = new CursorType ("ResizeUpDown");
+		public static readonly CursorType ResizeNESW = new CursorType("ResizeNESW");
+		public static readonly CursorType ResizeNWSE = new CursorType("ResizeNWSE");
 		public static readonly CursorType Hand = new CursorType ("Hand");
+		public static readonly CursorType OpenHand = new CursorType("OpenHand");
```

Comment:
We should rethink the naming here. We map `Hand` to `NSCursor.OpenHandCursor` on Mac already, so the naming is confusing and we can't simply swap them (could break Xwt users who expect the open hand cursor for `Hand`).

We should deprecate `Hand` (with `[Obsolete("Please use Hand1 or Hand2 instead")]`) and either use the Gtk style with `Hand1` and `Hand2`, or use `ClosedHand`, `OpenHand`. I'd prefer the first variant, cause we have only one hand cursor on Windows and the Gtk cursors are not necessarily `open`/`closed`, they're just two different hand styles.

Also the Cocoa `NSCursor.ClosedHandCursor` is mapped to `CursorType.Move` which makes sense. There is a third `NSCursor.PointingHandCursor` (not used right now) which would perfectly fit `CursorType.Hand2`.

|CursorType|Gtk|Cocoa|Wpf|
|---|---|---|---|
|`CursorType.Hand1`|`Gdk.CursorType.Hand1`|`NSCursor.OpenHandCursor`|`Cursors.Hand`|
|`CursorType.Hand2`|`Gdk.CursorType.Hand2`|`NSCursor.PointingHandCursor`|`Cursors.Hand`|